### ISSUE TITLE
Are the pixel means ordered by RGB or BGR?

### ISSUE THREAD

**Issue by bfialkoff** [mahyarnajibi/SNIPER#134]
I want to make an adjustment based on my dataset but its not clear if the code uses the standard rgb, or works with opencv's convention (bgr)

**Comment by zoe218** [comment]
hello,@bfialkoff,  Do you have a better performance on your dataset?

**Comment by bfialkoff** [comment]
No, I've given up. As far as I can see no one has managed to get good
results on their own dataset. It doesnt help that the config parameters
arent documented.
